swift_browser_ui_frontend/src/components/BrowserNavbar.vue

```diff
@@ -160,6 +163,19 @@ export default {
         "; path=/; expires="
         + expiryDate.toUTCString();
     },
+    changeActive (item) {
+      this.$store.commit("setActive", item);
+      let newParams = Object.fromEntries(Object.entries(
+        this.$route.params,
+      ));
+      if (newParams.project != undefined) {
```

Comment:
Is this for routes that don't have a project in the params?

Reply:
Mainly for the project selection in the navbar, for additional reason because of the upload runner – it cannot depend on the route as the user should be able do navigate the UI while uploading things.

Reply:
Project selector could use the route project id just as well, though